docs/ionq/jobs.md

```diff
@@ -0,0 +1,120 @@
+# Running IonQ API Jobs
+
+Here we detail how to run jobs on the IonQ API against the IonQ QPU and the
+IonQ simulator.
+
+In this section we assume a `cirq.ionq.Service` object has been instantiated and is
+called `service` and `cirq` and `cirq.ionq` have been imported:
+```python
+import cirq
+import cirq.ionq as ionq
+service = ionq.Service()
+```
+See [IonQ API Service](service.md) for how to set the service up.
+
+## Running programs
+
+The IonQ API is a service that allows you to send a quantum circuit as a *job*
+to a scheduler server.  This means that a user can submit a job to the API, and
+then this job is held in a queue before being scheduled to run on the appropriate
+hardware (QPU) or simulator.  Once a job is created (but not necessarily yet run)
+on the scheduler, the job is assigned an id and then one can query this
+job via the API. The job has a status on it which tells what state the job is in
+`running`, `completed`, `failed`, etc.  From a users perspective this is abstracted
+mostly away in Cirq, depending on whether one wants to make blocking calls (which
```

Comment:
`Depending on whether one wants to make blocking calls ` - > `Depending on whether you want to make blocking calls`

Reply:
done